pykafka/simpleconsumer.py

```diff
@@ -64,6 +64,7 @@ def __init__(self,
                  auto_commit_interval_ms=60 * 1000,
                  queued_max_messages=2000,
                  fetch_min_bytes=1,
+                 fetch_error_backoff_ms=500,
```

Comment:
I did as requested... and I'm having second thoughts. 
The discrepancy in parameters makes it harder to write code that just adapts to whichever class is chosen. My test harness becomes more complicated, and I fear it's typical. 
I propose reverting that change, but making it clear in the doc that the parameter is unused in all but the base case.
Thoughts?